api-contract/contract.yaml

```diff
@@ -512,7 +517,88 @@ paths:
         default:
           description: successful operation
           content: {}
-     
+  /notification:
+    get:
+      tags:
+      - notification
+      summary: Get notified when there is an interaction with users
+      description: This can be done by the logged in user.
+      operationId: notif
+      requestBody:
+        description: View notification object
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Notification'
+        required: true
+      security:
+        - bearer_auth: []
+      responses:
+        default:
+          description: successful operation
+          content: {}
+  /admin:
+    get:
+      tags:
+      - admin
+      summary: Be able to see user stats, posts, friends and groups
+      description: This can be done by the administrator.
+      operationId: seeStats
+      requestBody:
+        description: View admin object
+        content:
+          application/json:
+            schema:
+              type: array
+              $ref: '#/components/schemas/Admin'
+        required: true
```

Comment:
I believe this is not going to work. Check it in the OpenAPI preview, I bet it doesn't render as it should.